graylog2-web-interface/src/components/indices/StyledIndexSetDetailsRow.tsx

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (C) 2020 Graylog, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the Server Side Public License, version 1,
+ * as published by MongoDB, Inc.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Server Side Public License for more details.
+ *
+ * You should have received a copy of the Server Side Public License
+ * along with this program. If not, see
+ * <http://www.mongodb.com/licensing/server-side-public-license>.
+ */
+import styled from 'styled-components';
+
+import { Row } from 'components/graylog';
+
+const StyledIndexSetDetailsRow = styled(Row)`
```

Comment:
Something we should all discuss as a team, I don't mind having all the styles in their own specific file, but I want to make sure the whole team is onboard and we have a specific naming structure and some defined "conventions" (`IndexSetDetails.styles.tsx` maybe so related styles stay with their component)

Currently, styles have just been defined within the component file itself.